### PR TITLE
Update broken links in docs

### PR DESCRIPTION
**Pull request by pbojinov**
## Description

Links that previously pointed to [https://github.com/Shopify/web-foundation](https://github.com/Shopify/web-foundation) now redirect to [https://github.com/Shopify/web-configs](https://github.com/Shopify/web-configs) causing some of the documentation links to break.

It also looks like `web-foundation` was renamed to `web-foundations` (plural), causing an issue with some of the links.

## Type of change

This PR updates the links to use the new [https://github.com/Shopify/web-foundations](https://github.com/Shopify/web-foundations) documentation links.

## Checklist

- [ ] I have added a changelog entry, prefixed by the type of change noted above (Documentation fix and Test update does not need a changelog as we do not publish new version)

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ dev up
...
+We are adding documentation as we go in the [Web Foundation repo](https://github.com/Shopify/web-foundations). There you will find our [decision records](https://github.com/Shopify/web-foundations/tree/main/handbook/Decision%20Records), [principles](https://github.com/Shopify/web-foundations/tree/main/handbook/Principles), [best practices](https://github.com/Shopify/web-foundations/tree/main/handbook/Best%20Practices) and [styleguides](https://github.com/Shopify/web-foundations/blob/main/handbook/Styleguides) for writing and [testing](https://github.com/Shopify/web-foundations/blob/main/handbook/Best%20Practices/Testing.md) different kinds of components.
```
Comment (atesgoral) [inline .github/CONTRIBUTING.md:34]:
```suggestion
We are adding documentation as we go in the [Web Foundations repo](https://github.com/Shopify/web-foundations). There you will find our [decision records](https://github.com/Shopify/web-foundations/tree/main/handbook/Decision%20Records), [principles](https://github.com/Shopify/web-foundations/tree/main/handbook/Principles), [best practices](https://github.com/Shopify/web-foundations/tree/main/handbook/Best%20Practices) and [styleguides](https://github.com/Shopify/web-foundations/blob/main/handbook/Styleguides) for writing and [testing](https://github.com/Shopify/web-foundations/blob/main/handbook/Best%20Practices/Testing.md) different kinds of components.
```

File: documentation/getting-started.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+We are adding documentation as we go in the [Web Foundation repo](https://github.com/Shopify/web-foundations). There you will find our [decision records](https://github.com/Shopify/web-foundations/tree/main/handbook/Decision%20Records), [principles](https://github.com/Shopify/web-foundations/tree/main/handbook/Principles), [best practices](https://github.com/Shopify/web-foundations/tree/main/handbook/Best%20Practices) and [styleguides](https://github.com/Shopify/web-foundations/tree/main/handbook/Styleguides) for writing and [testing](https://github.com/Shopify/web-foundations/blob/main/handbook/Best%20Practices/Testing.md) different kinds of components.
```
Comment (atesgoral) [inline documentation/getting-started.md:5]:
```suggestion
We are adding documentation as we go in the [Web Foundations repo](https://github.com/Shopify/web-foundations). There you will find our [decision records](https://github.com/Shopify/web-foundations/tree/main/handbook/Decision%20Records), [principles](https://github.com/Shopify/web-foundations/tree/main/handbook/Principles), [best practices](https://github.com/Shopify/web-foundations/tree/main/handbook/Best%20Practices) and [styleguides](https://github.com/Shopify/web-foundations/tree/main/handbook/Styleguides) for writing and [testing](https://github.com/Shopify/web-foundations/blob/main/handbook/Best%20Practices/Testing.md) different kinds of components.
```

**Comment by atesgoral** [comment]
@pbojinov Thanks a lot for taking care of this!

**Comment by pbojinov** [comment]
@atesgoral my pleasure, glad I can help! Just added your suggestions.

**Review by gmcgibbon — APPROVED** [review]

**Review by atesgoral — APPROVED** [review]

**Pull request merged**